english/Rules to Success/Rules.txt

```diff
@@ -0,0 +1,17 @@
+Get comfortable being uncomfortable. Get outside your comfort zone on a regular basis. The more you do, the wider that area becomes. Things that used to seem impossible can become part of your everyday life.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.